docs/administration/enterprise.md

```diff
@@ -40,12 +40,15 @@ OpenCTI playbooks are flexible automation scenarios which can be fully customize
 
 ### Organizations management and segregation
 
-Organizations segregation is a way to segregate your data considering the organization associated to the users. Useful when your platform aims to share date to multiple organizations that share the access to the same OpenCTI platform. See [Organizations RBAC](../administration/users.md)
+Organizations segregation is a way to segregate your data considering the organization associated to the users. Useful when your platform aims to share date to multiple organizations that share the access to the same OpenCTI platform. Please read [Organizations RBAC](../administration/users.md) to get all information.
```

Comment:
to share **_data_** to multiple organization

Reply:
I would avoid to use to share twice.
Suggestion: Useful whe your platform aims to share data across multiple organizations that have access to the same OpenCTI platform.

Reply:
Please read [Organizations RBAC](../administration/users.md) to get all information.
- I would rather say "to get more information" but as you like
- I think the right link is "administration/segregation"